config/config.go

```diff
@@ -89,6 +89,55 @@ type MutableNodeConfig interface {
 		activeConsensusAlgo consensus.ConsensusAlgoType)
 }
 
+type BlockStorageConfig interface {
```

Comment:
I tried to keep config inside the services, but it would not work because of the circular dependencies needed to keep mutable config inside `config` package. Keeping that in mind the only way to have immutable config and have custom configuration per test group is to keep both config and its initialization inside of `config`.

We could have two copies of the same interface both in `config` and `services/blockstorage` but it would mean that we are just duplicating these definitions and I don't see how it's going to simplify development or improve maintainability.